src/main/resources/application.yaml

```diff
@@ -111,6 +111,8 @@ server:
   servlet:
     session:
       persistent: true
+transmissions:
+  snap-transmission-rate: ${SNAP_TRANSMISSION_RATE:PT5M}
```

Comment:
Not familiar with this syntax but It looks like this is setting the snap-transmission-rate to 5 minutes.

Is that right?

Reply:
Yup! "PT5M" is the default duration and it's in the "Duration" notation